resources/faq/why-are-remote-agents-not-updating.markdown

```diff
@@ -28,7 +28,7 @@ clients will see no need to scan for updates.
 
 * Check that the policy on in `$(sys.masterdir)` on the hub validates with
   `cf-promises`.
-* Check if `$(sys.inputdir)/cf_promsies_validated` differs from the
+* Check if `$(sys.inputdir)/cf_promises_validated` differs from the
```

Comment:
Did you git blame them? I bet they are all me!

Reply:
Nope, not really important to blame :)